lib/motsu-proc/src/test.rs

```diff
@@ -21,31 +21,49 @@ pub(crate) fn test(_attr: &TokenStream, input: TokenStream) -> TokenStream {
     }
 
     // Whether 1 or none contracts will be declared.
-    let contract_declarations = fn_args.into_iter().map(|arg| {
-        let FnArg::Typed(arg) = arg else {
-            error!(arg, "unexpected receiver argument in test signature");
-        };
-        let contract_arg_binding = &arg.pat;
-        let contract_ty = &arg.ty;
+    let arg_binding_and_ty = match fn_args
```

Comment:
We should probably be keeping a separate _CHANGELOG.md_ for `motsu` where we track its changes. This file would then be moved together with the crate to a separate repo.

Reply:
I would say we should start CHANGELOG for motsu in a new repository.